### PR TITLE
Contribuiting

### PR DESCRIPTION
**Pull request by aspacca**
Docs
## What does this PR do?
Add contributing guidelines

## Why is it important?
It gives contributors guidelines on the project

## Checklist

~- [ ] My code follows the style guidelines of this project~
~- [ ] I have commented my code, particularly in hard-to-understand areas~
- [x] I have made corresponding changes to the documentation
~- [ ] I have made corresponding change to the default configuration files~
~- [ ] I have added tests that prove my fix is effective or that my feature works~
~- [ ] I have added an entry in `CHANGELOG.md`~

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: CONTRIBUITING.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+   * `requirements-tests`   Install tests requirements on the host
+
+3. Run the linter, license check and test suite to ensure your changes do not break existing code. The `make` targets provided are the following:
+   * `lint`                 Lint the project on the host
+   * `black`                Run black in the project on the host
+   * `isort`                Run isort in the project on the host
+   * `mypy`                 Run mypy in the project on the host
+   * `license`              Run license validation in the project
+   * `test`                 Run all tests on the host
+   * `integration-test`     Run tests on docker
```
Comment (aspacca) [inline CONTRIBUITING.md:48]: `on the host`

File: CONTRIBUITING.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+   * `unit-test`            Run unit tests on the host
+   * `coverage`             Run tests on the host with coverage
+
+4. A subset of the previous tasks can be run in docker (that's the method used in CI), these are the equivalent `make` targe provided:
```
Comment (aspacca) [inline CONTRIBUITING.md:52]: `targets`

File: CONTRIBUITING.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+   * `docker-lint`          Lint the project on docker
+   * `docker-black`         Run black in the project on docker
+   * `docker-isort`         Run isort in the project on docker
+   * `docker-mypy`          Run mypy in the project on docker
+   * `docker-test`          Run tests on docker
+   * `docker-integration-test` Run tests on docker
```
Comment (aspacca) [inline CONTRIBUITING.md:58]: `Run integration tests on docker`

File: CONTRIBUITING.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+   * `docker-unit-test`     Run tests on docker
```
Comment (aspacca) [inline CONTRIBUITING.md:59]: `Run unit tests on docker`

File: CONTRIBUITING.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+   * `docker-coverage`      Run tests on docker with coverage
+
+5. Scripts for automated fixes of linting and license are provided where available. They are the following:
```
Comment (aspacca) [inline CONTRIBUITING.md:62]: `fixing`

File: CONTRIBUITING.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
+# Contributing to the Elastic Serverless Forwarder
+
+If you have a bugfix or new feature that you would like to contribute to
+elastic-serverless-forwarder, please find or open an issue about it first. Talk about what you would like to do. It may be that somebody is already working on it, or that there are particular issues that you should know about before implementing the change.
+
+We enjoy working with contributors to get their code accepted. There are many approaches to fixing a problem and it is important to find the best approach before writing too much code.
+
+## Running Elastic Serverless Forwarder locally
+
+We don't provide yet a tool for running Elastic Serverless Forwarder locally. A good first contribution would be to add such support.
```
Comment (kaiyan-sheng) [inline CONTRIBUITING.md:10]: Do we have an issue for this? 😬

Comment (aspacca) [reply CONTRIBUITING.md:10]: I will add one! 👍

Comment (aspacca) [reply CONTRIBUITING.md:10]: https://github.com/elastic/elastic-serverless-forwarder/issues/87

**Pull request merged**